apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/tour-card.js

```diff
@@ -2,7 +2,7 @@
  * External Dependencies
  */
 import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { getMediaQueryList } from '@automattic/viewport';
+import { getMediaQueryList, isMobile } from '@automattic/viewport';
```

Comment:
There's a way to highlight a bit more explicitly why we need to use `isMobile` here and not other utility function from `@automattic/viewport`.
See my comment on the original PR: https://github.com/Automattic/wp-calypso/pull/56347#pullrequestreview-828842013